tests/fixed/UselessConditions.php

```diff
@@ -103,12 +111,12 @@ public function uselessTernary() : bool
 
     public function uselessTernaryWithParameter(bool $condition) : bool
     {
-        return $condition;
+        return $condition ? true : false;
```

Comment:
See https://github.com/slevomat/coding-standard/issues/604 - CS doesn't know the type of `$condition` and in general it may produce broken code (the method would return non-bool). This is now still reported, but not auto-fixed.